doc/source/user_guide/missing_data.rst

```diff
@@ -832,6 +832,12 @@ return ``False``.
    pd.NA == pd.NA
    pd.NA < 2.5
 
+There are a few special cases when the result is known, even when one of the
```

Comment:
I would maybe move this above the equality/comparison operations (as this belongs to the arithmetic operations, and the isna(..) also follows more closely on comparison operations then)